web/app/Makefile

```diff
@@ -0,0 +1,3 @@
+.PHONY: dev
+dev:
+	BABEL_ENV=development yarn webpack-dev-server
```

Comment:
we'll need to set `BABEL_ENV` in `web/Dockerfile` as well

also, can you update the instructions in BUILD.md to include updated running instructions?